src/automation/azext_automation/aaz/latest/automation/hrwg/_update.py

```diff
@@ -1,384 +1,407 @@
-    hybrid_runbook_worker_group_read.id = AAZStrType(
-        flags={"read_only": True},
-    )
-    hybrid_runbook_worker_group_read.name = AAZStrType(
-        flags={"read_only": True},
-    )
-    hybrid_runbook_worker_group_read.properties = AAZObjectType(
-        flags={"client_flatten": True},
-    )
-    hybrid_runbook_worker_group_read.system_data = AAZObjectType(
-        serialized_name="systemData",
-        flags={"read_only": True},
-    )
-    hybrid_runbook_worker_group_read.type = AAZStrType(
-        flags={"read_only": True},
-    )
-
-    properties = _schema_hybrid_runbook_worker_group_read.properties
-    properties.credential = AAZObjectType()
-    properties.group_type = AAZStrType(
-        serialized_name="groupType",
-    )
-
-    credential = _schema_hybrid_runbook_worker_group_read.properties.credential
-    credential.name = AAZStrType()
-
-    system_data = _schema_hybrid_runbook_worker_group_read.system_data
-    system_data.created_at = AAZStrType(
-        serialized_name="createdAt",
-        flags={"read_only": True},
-    )
-    system_data.created_by = AAZStrType(
-        serialized_name="createdBy",
-        flags={"read_only": True},
-    )
-    system_data.created_by_type = AAZStrType(
-        serialized_name="createdByType",
-        flags={"read_only": True},
-    )
-    system_data.last_modified_at = AAZStrType(
-        serialized_name="lastModifiedAt",
-        flags={"read_only": True},
-    )
-    system_data.last_modified_by = AAZStrType(
-        serialized_name="lastModifiedBy",
-        flags={"read_only": True},
-    )
-    system_data.last_modified_by_type = AAZStrType(
-        serialized_name="lastModifiedByType",
-        flags={"read_only": True},
-    )
-
-    _schema.id = _schema_hybrid_runbook_worker_group_read.id
-    _schema.name = _schema_hybrid_runbook_worker_group_read.name
-    _schema.properties = _schema_hybrid_runbook_worker_group_read.properties
-    _schema.system_data = _schema_hybrid_runbook_worker_group_read.system_data
-    _schema.type = _schema_hybrid_runbook_worker_group_read.type
-
-
-__all__ = ["Update"]
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "automation hrwg update",
+)
+class Update(AAZCommand):
+    """Update a hybrid runbook worker group.
+
+    :example: Update hybrid worker group
+        az automation hrwg update --automation-account-name accountName --resource-group groupName  --name hybridrunbookworkergroupName --credential "{name: credentialname}"

```

Comment:
```suggestion
        az automation hrwg update --automation-account-name accountName --resource-group groupName --name hybridrunbookworkergroupName --credential "{name: credentialname}"

```

Reply:
updated

Reply:
@kairu-ms I find it easy to mistype extra spaces when he enters examples by Workspace Editors.
I'm not sure if this is a common problem. If so, could we trim the user's input to eliminate the extra spaces in Workspace Editors?

Reply:
I've added a new task in CodeGen V2 project plan. We will fix it in when work on examples validation.